packages/nx/src/tasks-runner/life-cycles/get-read-from-cache-lines.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { EOL } from 'os';
+
+import { output } from '../../utils/output';
+import { getReadFromCacheLines } from './get-read-from-cache-lines';
+
+describe('getReadFromCacheLines', () => {
+  it.each([
```

Comment:
Can we split these up into individual `it` blocks? That should feel more cohesive with the rest of the codebase.